modules/google-analytics/wp-google-analytics.php

```diff
@@ -0,0 +1,615 @@
+				array(
+						'token'            => '%the_author%',
+						'callback'         => 'get_the_author',
+						'callback_returns' => 'string',
+						'description'      => __( 'Post author for current view', 'wp-google-analytics' ),
+						'retval'           => __( "Post author's display name", 'wp-google-analytics' ),
+						'ignore_when'      => array(
+								'is_home',
+								'is_front_page',
+								'is_post_type_archive',
+								'is_page',
+								'is_date',
+								'is_category',
+								'is_tag',
+							),
+					),
+				array(
+						'token'            => '%the_category%',
+						'callback'         => array( $this, 'token_the_category' ),
+						'callback_returns' => 'string',
+						'description'      => __( 'Categories assigned to a post', 'wp-google-analytics' ),
+						'retval'           => __( "Category names in a commma-separated list", 'wp-google-analytics' ),
+						'ignore_when'      => array(
+								'is_home',
+								'is_front_page',
+								'is_page',
+								'is_post_type_archive',
+								'is_author',
+								'is_tag',
+							),
+					),
+				array(
+						'token'            => '%context%',
+						'callback'         => array( $this, 'token_context' ),
+						'callback_returns' => 'string',
+						'description'      => __( 'Which view the visitor is on', 'wp-google-analytics' ),
+						'retval'           => __( "Samples: 'home', 'category', 'post', 'author'" ),
+					),
+				array(
+						'token'            => '%the_date%',
+						'callback'         => 'get_the_date',
+						'callback_returns' => 'string',
+						'description'      => __( 'Publication date for the current view', 'wp-google-analytics' ),
+						'retval'           => __( "Format specified by 'Date Format' in Settings -> General", 'wp-google-analytics' ),
+						'ignore_when'      => array(
+								'is_home',
+								'is_front_page',
+								'is_post_type_archive',
+								'is_page',
+								'is_author',
+								'is_category',
+								'is_tag',
+							),
+					),
+				array(
+						'token'            => '%the_tags%',
+						'callback'         => array( $this, 'token_the_tags' ),
+						'callback_returns' => 'string',
+						'description'      => __( 'Tags assigned to a post', 'wp-google-analytics' ),
+						'retval'           => __( "Tag names in a commma-separated list", 'wp-google-analytics' ),
+						'ignore_when'      => array(
+								'is_home',
+								'is_front_page',
+								'is_page',
+								'is_post_type_archive',
+								'is_date',
+								'is_category',
+								'is_author',
+							),
+					),
+				array(
+						'token'            => '%is_user_logged_in%',
+						'callback'         => 'is_user_logged_in',
+						'callback_returns' => 'bool',
+						'description'      => __( 'Whether or not the viewer is logged in', 'wp-google-analytics' ),
+						'retval'           => __( "'true' or 'false'", 'wp-google-analytics' ),
+					),
+			);
+
+		$this->tokens = apply_filters( 'wga_tokens', $this->tokens );
+	}
+
+	/**
```

Comment:
I agree.  @rodrigoi Can we remove this extra settings code here? It is going to be managed in Calypso.